searchcore/src/vespa/searchcore/proton/matching/blueprintbuilder.cpp

```diff
@@ -67,7 +67,8 @@ class BlueprintBuilderVisitor :
     void buildIntermediate(IntermediateBlueprint *b, NodeType &n) __attribute__((noinline));
 
     void buildWeakAnd(ProtonWeakAnd &n) {
-        auto *wand = new WeakAndBlueprint(n.getTargetNumHits(), _requestContext.get_attribute_blueprint_params().weakand_range);
+        auto *wand = new WeakAndBlueprint(n.getTargetNumHits(), _requestContext.get_attribute_blueprint_params().weakand_range,
+                                          _requestContext.thread_bundle().size() > 1);
```

Comment:
should use use is_search_multi_threaded here as well

Reply:
CreateBlueprintVisitorHelper was not used here, but made similar method to unify code.